apps/api/src/persistence/database/types/ArangoCollectionId.ts

```diff
@@ -15,10 +15,17 @@ export const categoryEdgeCollectionID = 'category_edges';
 
 export const edgeConnectionCollectionID = 'resource_edge_connections';
 
-export const arangoEdgeCollectionIDs = [categoryEdgeCollectionID, edgeConnectionCollectionID];
+export const arangoEdgeCollectionIDs = [
+    categoryEdgeCollectionID,
+    edgeConnectionCollectionID,
+] as const;
+
+export type ArangoEdgeCollectionID = typeof arangoEdgeCollectionIDs[number];
 
-export const isArangoEdgeCollectionCollectionID = (input: unknown): boolean =>
-    arangoEdgeCollectionIDs.includes(input as string);
+export const isArangoEdgeCollectionCollectionID = (
```

Comment:
Let's bump the refactor of Arango category management in priority as far as tech-debt stories go.

Reply:
Done